distribution/sdk/artemis-sandbox/src/main/java/sdk.example.route/AmqpProducerConsumerExample.java

```diff
@@ -57,7 +57,7 @@ public void configure() throws Exception {
     onException(Throwable.class).logStackTrace(true);
 
     createCamelContext();
-    CamelContext camelContext = getContext();
+
```

Comment:
❓ Wasn't the whole point of this example to show how to create a camel context and retrieve it? Should we keep this and show how to use it or at least keep the lines commented?